crates/pallets/starknet/src/genesis_loader.rs

```diff
@@ -38,10 +38,21 @@ type StorageKey = HexFelt;
 type ContractStorageKey = (ContractAddress, StorageKey);
 type StorageValue = HexFelt;
 
+/// A struct containing predeployed accounts info.
+#[derive(Serialize, Deserialize)]
+pub struct PredeployedAccount {
+    pub contract_address: FieldElement,
+    pub class_hash: FieldElement,
+    pub name: String,
+    pub private_key: Option<Vec<u8>>,
+    // pub balance: FieldElement,
```

Comment:
Why commented?

Reply:
That's an artifact from a previous commit, this attribute iis never used. 

The balance of each predeployed account is read using the `call` method from `StarknetReadRpcApi` at the latest block tag.

Reply:
So you can remove it

Reply:
yes working on it